tool/node

```diff
@@ -49,7 +49,8 @@ fi
                 ARCH="arm64"
             fi
             mkdir -p "$cachedir"
-            curl -f -L -o "$tarball" "https://nodejs.org/dist/v${want_rev}/node-v${want_rev}-${OS}-${ARCH}.tar.gz"
+	    # Running behind a load balancer can cause problems with HTTP/2
```

Comment:
Can you elaborate? Or say that this is an experiment and speculative?

Reply:
I googled the error message and "load balancers don't handle HTTP/2 quite right" was the most common answer for what causes INTERNAL_ERROR messages and switching to HTTP 1.1 was the most common solution. It seems to work and I wanted to spend an appropriate amount of time on this issue so I didn't try to understand in more detail. What do you think is an appropriate level of documentation here?

Reply:
This comment as-is doesn't give any information about when we can remove this hack or any clues to figure that out.

Also, it's more confident sounding that the PR description itself which says that this might be the answer.

Ideally there would be a bug showing the exact error we got without this, and in which environment, and which curl version, and pointing the finger at a specific load balancer/host/vendor that we can then work with to get fixed (I worked with a handful of vendors when I wrote Go's HTTP/2 implementation). Then people reading this comment can look at the issue and see if it's been resolved and know they can then remove the flag.

Reply:
(and then once you have a bug open, replace `#cleanup` in the commit message with the number)

Reply:
Okay! Here you go: https://github.com/tailscale/tailscale/issues/8988

Let me know what edits you have. Thanks!